### ISSUE TITLE
Code comment in spec layout hard to read / accessibility contrast issue

### ISSUE THREAD

**Issue by hadro** [IIIF/website#699]
From @azaroth42 , "grey comment in JSON on black background is impossible to read, accessibility issue" -- see e.g. https://preview.iiif.io/api/search-v3-alignment/search/2.0/#simple-lists
